src/scenes/home/gala/gala.css

```diff
@@ -1,38 +1,15 @@
-.header {
+/*Hero*/
+.hero {
+  background-image: url('./portland19.jpg');
+  height: 100vh;
   text-align: center;
-  padding-bottom: 20px;
 }
-
-.cta {
-  text-align: center;
-}
-
-.tickets {
-  padding-top: 15px;
-  display: flex;
-  flex-flow: row wrap;
-  justify-content: space-around;
-}
-
-.tickets > div {
-  margin-bottom: 20px;
-  display: inline-block;
-  width: 45%;
-  max-width: 560px;
-}
-
-@media screen and (min-width: 415px) and (max-width: 1280px) {
-  .tickets > div {
-    width: 375px;
-  }
-}
-
-@media screen and (max-width: 414px) {
-  .tickets > div {
-    width: 300px;
-  }
+.heading {
+  color: #fff;
+  padding: 10% 0;
 }
 
-iframe {
-  border: 0;
+a {
+  color: #fff;
+  font-size: 1.2em;
 }
```

Comment:
We can't allow CSS styles directly on elements, as this will cause unintended side effects elsewhere in the code. Please replace this with a class rule.